src/azure-cli/azure/cli/command_modules/acr/policy.py

```diff
@@ -4,35 +4,42 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core.commands import LongRunningOperation
-from ._utils import validate_managed_registry
+from ._utils import validate_premium_registry
 
 
-POLICIES_NOT_SUPPORTED = 'Policies are only supported for managed registries.'
+POLICIES_NOT_SUPPORTED = 'Policies are only supported for managed registries in Premium SKU.'
 
 
 def acr_config_content_trust_show(cmd,
-                                  client,
                                   registry_name,
                                   resource_group_name=None):
-    _, resource_group_name = validate_managed_registry(
+    registry, _ = validate_premium_registry(
```

Comment:
Did we ever allow non-premium registries with policy? Is this a breaking change?

Reply:
Nope, premium was always needed.